packages/node/test/transports/http.test.ts

```diff
@@ -27,11 +28,11 @@ function createTransport(options: TransportOptions): HTTPTransport {
 }
 
 function assertBasicOptions(options: any): void {
-  expect(options.headers['X-Sentry-Auth']).toContain('sentry_version');
-  expect(options.headers['X-Sentry-Auth']).toContain('sentry_client');
-  expect(options.headers['X-Sentry-Auth']).toContain('sentry_key');
+  // expect(options.headers['X-Sentry-Auth']).toContain('sentry_version');
+  // expect(options.headers['X-Sentry-Auth']).toContain('sentry_client');
+  // expect(options.headers['X-Sentry-Auth']).toContain('sentry_key');
```

Comment:
If we're okay with sending auth via query string in node, then we can get rid of these lines.

The downside of auth in the query string is increased likelihood of having the "public key" part of the DSN exposed in logs, etc. Since this is more like "identification" then real "authentication", I believe it is not a huge concern, while allowing us to have simpler code.